defaults/main.yml

```diff
@@ -10,10 +10,10 @@ storage_pool_defaults:
   type: lvm
 
   encryption: false
-  encryption_passphrase: null
-  encryption_key_file: null
+  private_passphrase: null
+  private_key: null
```

Comment:
"private" does not seem to make sense in this context:

1) passphrases are always private and it is not the passphrase for a private key
2) the key is not an assymmetric private key but just a (ideally randomly generated) binary blob

To group things it might make more sense to do:

For disabled encryption
```yaml
encryption: None
```

For enabled encryption:
```yaml
encryption:
    passphrase: ...
    key: ...
```

Reply:
not changing `encryption` to be a `dict` value as that would involve changing more code

however, using `key` instead of `encryption_key_file` and using `password` instead of `encryption_passphrase`